.rspec

```
@@ -1,2 +1,3 @@
 --color
 --require spec_helper
+--fail-fast
```

Its good practice to actually run the test files directly rather than use fail fast. I'll leave this to you!